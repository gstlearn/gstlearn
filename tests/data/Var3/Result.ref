
Variogram characteristics
=========================
Number of variable(s)       = 1
Number of direction(s)      = 1
Space dimension             = 3
Variable(s)                 = [Unknown]

Variance-Covariance Matrix     0.994

Direction #1
------------
Number of lags              = 10
Direction coefficients      =      0.000     1.000     0.000
Direction angles (degrees)  =     90.000     0.000     0.000
Tolerance on direction      =      0.500 (degrees)
Calculation lag             =      0.900
Tolerance on distance       =     50.000 (Percent of the lag value)

For variable 1
      Rank    Npairs  Distance     Value
         1   768.000     1.000     0.488
         2  2830.000     1.964     0.704
         3  1334.000     2.916     0.770
         4  3162.000     3.504     0.798
         5  2940.000     4.283     0.807
         6  5402.000     5.314     0.819
         7  4490.000     6.324     0.802
         8  4592.000     7.234     0.863
         9  5008.000     8.154     0.893

Model characteristics
=====================
Space dimension              = 3
Number of variable(s)        = 1
Number of basic structure(s) = 3
Number of drift function(s)  = 0
Number of drift equation(s)  = 0

Covariance Part
---------------
Cubic
- Sill         =      0.422
- Range        =      3.350
Exponential
- Sill         =    136.035
- Range        =  22275.157
- Theo. Range  =   7435.631
Spherical
- Sill         =      0.302
- Range        =      0.980
Total Sill     =    136.759
Known Mean(s)     0.000
Goodness-of-fit (as a percentage of the variance) corresponds to level #1 (1 for very good)
