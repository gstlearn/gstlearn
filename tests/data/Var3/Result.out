
Model characteristics
=====================
Space dimension              = 3
Number of variable(s)        = 1
Number of basic structure(s) = 3
Number of drift function(s)  = 0
Number of drift equation(s)  = 0

Covariance Part
---------------
Nugget Effect
- Sill         =      0.193
Cubic
- Sill         =  29761.625
- Range        =  11081.873
Spherical
- Sill         =      0.575
- Range        =      2.815
Total Sill     =  29762.393
